webpack/ForemanTasks/Components/TaskActions/UnlockModals.js

```diff
@@ -22,12 +22,12 @@ export const UnlockModal = ({ onClick, id }) => (
   />
 );
 
-export const ForceUnlockModal = ({ onClick, id }) => (
+export const ForceUnlockModal = ({ onClick, id, selectedRowsLen }) => (
   <ClickConfirmation
     id={id}
     title={__('Force Unlock')}
     body={__(
-      'Resources will be unlocked and will not prevent other tasks from being run. As the task might be still running, it should be avoided to use this unless you are really sure the task got stuck'
+      `Resources for ${selectedRowsLen} task(s) will be unlocked and will not prevent other tasks from being run. As the task might be still running, it should be avoided to use this unless you are really sure the task got stuck.`
```

Comment:
I don't think the translation work that way, see: http://theforeman.github.io/foreman/?path=/docs/introduction-internationalization--page

Reply:
You're right, thanks!